autotest/test_par_gwf_ims_csv.py

```diff
@@ -0,0 +1,263 @@
+import os
+
+import flopy
+import numpy as np
+import pytest
+from framework import TestFramework
+from simulation import TestSimulation
+
+# Test for parallel MODFLOW running on two cpus.
+# It contains two coupled models with
+#
+# 1d:  (nlay,nrow,ncol) = (1,1,5),
+#
+# constant head boundaries left=1.0, right=10.0.
+# The result should be a uniform flow field.
+
+ex = ["par_gwf_csv"]
+dis_shape = [(1, 1, 5)]
+
+# global convenience...
+name_left = "leftmodel"
+name_right = "rightmodel"
+
+# solver data
+nouter, ninner = 100, 300
+hclose, rclose, relax = 10e-6, 1e-3, 0.97
+
+
+def get_model(idx, dir):
```

Comment:
If this is the same (or nearly the same) as what's in `test_par_gwf01.py` is there any way you can import `build_model` from that python file and not reproduce here?  Maybe more trouble than it's worth for now, but might be nice to move in that direction.